File: problems/longest-common-subsequence/boilerplate-full/function.js

  ##USER_CODE_HERE##
  
  const input = require('fs').readFileSync('/dev/stdin', 'utf8').trim().split('\n').join(' ').split(' ');
  const str1 = parseInt(input.shift());
  const str2 = parseInt(input.shift());
  const result = longestCommonSubsequence(str1, str2);
  console.log(result);
      

File: problems/longest-common-subsequence/boilerplate-full/function.rs

  use std::io::{self, BufRead};
  
  ##USER_CODE_HERE##
  
  fn main() {
    let stdin = io::stdin();
    let mut input = stdin.lock().lines().map(|l| l.unwrap());
    let str1: String = input.next().unwrap().parse().unwrap();
  let str2: String = input.next().unwrap().parse().unwrap();
    let result = longestCommonSubsequence(str1, str2);
    println!("{}", result);
  }
      